diff --git a/subprojects/frida-core/server/server.vala b/subprojects/frida-core/server/server.vala
--- a/subprojects/frida-core/server/server.vala
+++ b/subprojects/frida-core/server/server.vala
@@ -1,6 +1,6 @@
 namespace Frida {
 	public class Server : Object {
-		public uint port { get; construct; default = 27042; }
+		public uint port { get; construct; default = 27088; }
 		public string? address { get; construct; }
 
 		public signal void expired ();
diff --git a/subprojects/frida-gum/gum/gumjs/gumscriptbackend.c b/subprojects/frida-gum/gum/gumjs/gumscriptbackend.c
--- a/subprojects/frida-gum/gum/gumjs/gumscriptbackend.c
+++ b/subprojects/frida-gum/gum/gumjs/gumscriptbackend.c
@@ -50,7 +50,7 @@
-      "gum-js-loop",
+      "core-js-loop",
diff --git a/subprojects/frida-gum/gum/backend-darwin/gummemory-darwin.c b/subprojects/frida-gum/gum/backend-darwin/gummemory-darwin.c
--- a/subprojects/frida-gum/gum/backend-darwin/gummemory-darwin.c
+++ b/subprojects/frida-gum/gum/backend-darwin/gummemory-darwin.c
@@ -37,7 +37,7 @@
-  "frida-agent.dylib",
+  "internal-agent.dylib",
diff --git a/subprojects/frida-core/lib/base/session.vala b/subprojects/frida-core/lib/base/session.vala
--- a/subprojects/frida-core/lib/base/session.vala
+++ b/subprojects/frida-core/lib/base/session.vala
@@ -1,6 +1,6 @@
-		public string p_tag { get; default = "frida"; }
+		public string p_tag { get; default = "core"; }
